packages/playwright-core/src/client/connection.ts

```diff
@@ -110,8 +110,8 @@ export class Connection extends EventEmitter {
   }
 
   async sendMessageToServer(object: ChannelOwner, method: string, params: any, apiName: string | undefined, frames: channels.StackFrame[], wallTime: number | undefined): Promise<any> {
-    if (this._closedErrorMessage)
-      throw new Error(this._closedErrorMessage);
+    if (this._closedError)
+      throw this._closedError;
```

Comment:
This has an effect of giving the stack of the error rather than the stack of this call. Not sure this is intentional.

Reply:
As this call is always wrapped, stack will be rewritten.